server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -143,8 +145,10 @@ object CORS {
       F: Applicative[F]): Http[F, G] =
     Kleisli { req =>
       // In the case of an options request we want to return a simple response with the correct Headers set.
-      def createOptionsResponse(origin: Header.Raw, acrm: Header.Raw): Response[G] =
-        corsHeaders(origin.value, acrm.value, isPreflight = true)(Response())
+      def createOptionsResponse(
+          origin: Origin,
+          acrm: `Access-Control-Request-Method`): Response[G] =
```

Comment:
I'm surprised this didn't offend MiMa.  Does it not check internal functions?  I think it's safe.